.travis.yml

```diff
@@ -6,6 +6,10 @@ sudo: false
 # Make sure we are on Ubuntu 14.04
 dist: trusty
 
+branches:
+  except:
+    - /pyup-.*/
```

Comment:
Pyup creates a branch and pull request per requirement that needs updating. If the pyup branches aren't excluded every time a pull request is opened two to three jobs are created on Travis and AppVeyor. Excluding the pyup branches removes all but the pull request job which is what is wanted.